### ISSUE TITLE
Can't duplicate mother tab without duplicating children

### ISSUE THREAD

**Issue by bsilvern** [piroor/treestyletab#1172]
## Short description
Using <CTRL>drag to duplicate child tabs works as expected. But doing that to the mother tab when there are several child tabs causes the entire tree to be duplicated. There seems to be no way to duplicate only the mother tab.

## Steps to reproduce

 1. Start Filrefox with clean profile.
 2. Install TST.
 3. Create a mother tab with several child tabs.
 4. Try to duplicate the mother tab using <CTRL>drag.

## Expected result
Only the mother tab is duplicated.

## Actual result
All tabs in the tree are duplicated. Furthermore, the tree structure of the duplicated tabs is lost with all tabs becoming top-level. 

Related feature request: It would be nice to be able to <CTRL>click on any tab to duplicate it.

## Environment

 * Platform (OS): Windows 10
 * Version of Firefox: 48.0
 * Version (or revision) of Tree Style Tab: 0.17.2016061501

**Comment by piroor** [comment]
Right-click on a parent tab and choose "Duplicate Tab" (the menu is provided by the Firefox itself), then you'll get just one duplicated tab.

Dragging of a parent tab produces moving of the whole tree. I is a basic behavior of TST, and I think it is natural. And, if you drop the tree onto a bookmark folder, then multiple bookmarks are created there from the tree. I think it is also a naturally expected result. (I think they bookmarks should be created in a new folder, but it is a different issue.) Thus, Ctrl-drag of a parent tab is also expected to duplicate the whole tree. Adding new option to disable this feature is easy, but I don't want to increase the number of options for unnatural behaviors - I think that TST should have options only for some edge cases which have multiple possible natural behaviors. Sorry.

> Furthermore, the tree structure of the duplicated tabs is lost with all tabs becoming top-level. 

Hmm, it needs to be solved. As I described above, duplicated tree should keep its structure from the original.

**Comment by bsilvern** [comment]
When I right click on a tab in Firefox (with or without TST) I do not see a "Duplicate Tab" option.

**Comment by piroor** [comment]
It is a built-in feature of Firefox. Did you hide it by Tab Mix Plus, Menu Editor, or something other addon?

**Comment by bsilvern** [comment]
I tried it it with all add-ons disabled with the same result -- No "Duplicate Tab".
Tired it on another PC -- same thing.

See https://support.mozilla.org/en-US/questions/762043. No mention of a right-click duplicate option.

I did however learn that there are other ways to accomplish duplicating the mother tab, such as <CTRL>clicking the reload button. When done with a a TST mother tab active , that does duplicate the tab, creating a new top-level tab. I think creating a child tab would be the more desired behavior, but I guess that's a preference.

**Comment by piroor** [comment]
I'm very sorry. My "Duplicate Tab" was added by Multiple Tab Handler. Firefox still has its own duplicate tab command internally but there is no UI...

**Comment by bsilvern** [comment]
No worries. I'm just glad I now know the trick for duping.
Thanks for the prompt replies.

**Comment by piroor** [comment]
> I did however learn that there are other ways to accomplish duplicating the mother tab, such as clicking the reload button. When done with a a TST mother tab active , that does duplicate the tab, creating a new top-level tab. I think creating a child tab would be the more desired behavior, but I guess that's a preference.

TST has an option for this command. See the "New Tabs" pane in the configuration please.

**Comment by bsilvern** [comment]
Thanks! That works perfectly.
